lib/network/services/breeds_api_service.dart

```diff
@@ -14,4 +14,14 @@ class BreedsApiService {
             BreedResponse.fromJson(element as Map<String, dynamic>))
         .toList();
   }
+  Future<List<BreedResponse>> getPhotos(String id) async {
```

Comment:
Make as getter in class to reuse it when you will have new methods with api calls. 
`const _apiKey = 'live_n4d5FPV0uY0Br4loCo13WbraocoWtDv8jm3zlqw4iCvmShOAuxT9NDHeaUwYcEoB';`